core/unit_test/CMakeLists.txt

```
@@ -337,12 +337,14 @@ FUNCTION (KOKKOS_ADD_INCREMENTAL_TEST BACK_END_NAME)
     IF (NOT CURRENT_TEST_NAME IN_LIST Kokkos_${BACK_END_NAME}_EXCLUDE_TESTS)
        SET (CURRENT_TEST_OUTPUT_FILENAME ${CURRENT_TEST_NAME}_${BACK_END_NAME})
        FILE( STRINGS ${CURRENT_FILE_PATH} CURRENT_REQUIRED_FEATURE_LINE REGEX "Kokkos_Feature_Level_Required" ) 
+       if (NOT ${CURRENT_REQUIRED_FEATURE_LINE} STREQUAL "") 
```

```suggestion
       IF (NOT ${CURRENT_REQUIRED_FEATURE_LINE} STREQUAL "") 
```
As much as I prefer lowercase for CMake, don't break consistency.

Also why did you change this?  It does not feel like a good idea because if one omit or misspell "Kokkos_Feature_Level_Required", the unit test will be silently discarded.